src/renderer/cursor_renderer/mod.rs

```diff
@@ -32,54 +32,24 @@ pub struct CursorSettings {
     vfx_particle_curl: f32,
 }
 
-pub fn initialize_settings() {
-    SETTINGS.set(&CursorSettings {
-        antialiasing: true,
-        animation_length: 0.13,
-        animate_in_insert_mode: true,
-        trail_size: 0.7,
-        vfx_mode: cursor_vfx::VfxMode::Disabled,
-        vfx_opacity: 200.0,
-        vfx_particle_lifetime: 1.2,
-        vfx_particle_density: 7.0,
-        vfx_particle_speed: 10.0,
-        vfx_particle_phase: 1.5,
-        vfx_particle_curl: 1.0,
-    });
-
-    register_nvim_setting!("cursor_antialiasing", CursorSettings::antialiasing);
-    register_nvim_setting!(
-        "cursor_animate_in_insert_mode",
-        CursorSettings::animate_in_insert_mode
-    );
-    register_nvim_setting!("cursor_animation_length", CursorSettings::animation_length);
-    register_nvim_setting!("cursor_trail_size", CursorSettings::trail_size);
-    register_nvim_setting!("cursor_vfx_mode", CursorSettings::vfx_mode);
-    register_nvim_setting!("cursor_vfx_opacity", CursorSettings::vfx_opacity);
-    register_nvim_setting!(
-        "cursor_vfx_particle_lifetime",
-        CursorSettings::vfx_particle_lifetime
-    );
-    register_nvim_setting!(
-        "cursor_vfx_particle_density",
-        CursorSettings::vfx_particle_density
-    );
-    register_nvim_setting!(
-        "cursor_vfx_particle_speed",
-        CursorSettings::vfx_particle_speed
-    );
-    register_nvim_setting!(
-        "cursor_vfx_particle_phase",
-        CursorSettings::vfx_particle_phase
-    );
-    register_nvim_setting!(
-        "cursor_vfx_particle_curl",
-        CursorSettings::vfx_particle_curl
-    );
+impl Default for CursorSettings {
```

Comment:
This is WAY cleaner. Seems like it enforces a naming scheme for the resulting option names inside of neovim?